src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -29,15 +29,27 @@ class DebugCommand extends Command
     private $twig;
 
     /**
-     * {@inheritdoc}
+     * @param Environment $twig
      */
     public function __construct($name = 'debug:twig')
```

Comment:
i preserved the sig for BC.. but thas from a reflection sniffing perspective. Which is nonsense i guess :)

`__construct($twig = null` is is then :+1: